tiledb/sm/filesystem/win.cc

```diff
@@ -77,10 +79,28 @@ std::string get_last_error_msg() {
     LocalFree(lpMsgBuf);
     return "unknown error";
   }
-  std::string msg(reinterpret_cast<char*>(lpMsgBuf));
+  LPVOID lpDisplayBuf;
+  lpDisplayBuf = (LPVOID)LocalAlloc(
```

Comment:
Is there a particular reason to use a Windows system allocator rather than a C++ allocator? Other than that it's feeding another one next?

Reply:
A) it was in the example I referenced
B) its the same as used by the formatmessage(), and thus a bit more uniform within the routine